app/templates/layouts/asylum-base.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="charset" content="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>{% block page_title %}{{ config.APP_NAME }}{% endblock %}</title>
+
+        {% assets 'asylum_css' %}<link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">{% endassets %}
+        {% assets 'asylum_scss' %}<link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">{% endassets %}
+
+        <!-- Google Translate -->
+        <script src="http://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans:600' rel='stylesheet' type='text/css'>
+
+        {% assets 'vendor_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
+        {% assets 'app_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
+        {% assets 'asylum_js' %}<script type="text/javascript" src="{{ ASSET_URL }}"></script>{% endassets %}
+    </head>
+
+    <body>
+        {% include 'partials/_flashes.html' %}
```

Comment:
We use this on our other flask pages. Just by default we give every page a link to the flashes partial in case we need to flash information to the user. The "flash" is basically just a message (like `invalid password` or `unknown username`) that gets shown to the user after an error occurs on the server on their next pageload (but not after that). So usually it's used on, like, login forms to say "whoops! passwords need to be >6 characters, please try again!" when you try to make an account.

I leave this here even though it *probably* won't be used (i don't see why we'd have a flash on the homepage) so that if we ever DO need to use it (maybe we have the ability to add or report resources from the homepage or something and errors can occur that require a flash) we don't end up spending time pointlessly debugging when the issue is simply that we don't have this line of code.

It does nothing right now, but could potentially save a headache in the future.